src/sdk.ts

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) 2023, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import fs from 'fs'
+import path from 'path'
+
+export function getVersionFromSdk(sdkPath: string): string | null {
+  const versionFilePath = path.join(sdkPath, 'version')
+
+  if (fs.existsSync(versionFilePath)) {
+    return fs.readFileSync(versionFilePath, 'utf8').trim()
+  } else {
+    return null
```

Comment:
Should we throw here? Or is there a legit case where there is no `version` file?

The action will end up storing an empty string in the output in this case:
https://github.com/actions/toolkit/blob/0db3029fcfcb1393a2b3ccd14caecd7a3460af4e/packages/core/src/utils.ts#L11